sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOStorageReadTest.java

```diff
@@ -1493,6 +1399,17 @@ public void testReadFromBigQueryIO() throws Exception {
                 StreamPosition.newBuilder().setStream(Stream.newBuilder().setName("streamName")))
             .build();
 
+    /*
```

Comment:
Should this be here?

Reply:
Was using it as a reference when debugging. Should be deleted now.
Thanks!